recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class ZmqppConan(ConanFile):
+    name = "zmqpp"
+    homepage = "https://github.com/zeromq/zmqpp"
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "This C++ binding for 0mq/zmq is a 'high-level' library that hides most of the c-style interface core 0mq provides."
+    topics = ("conan", "zmq", "0mq", "zeromq", "message-queue", "asynchronous")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = ["cmake", "cmake_find_package"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("libsodium/1.0.18")
```

Comment:
Why this lib is in requirements? Doesn't seem to be a direct dependency.

Reply:
i think there was some issues with macOS build some time ago, but I'll try without it.